services/libs/aster-std/src/fs/utils/inode.rs

```diff
@@ -364,23 +360,23 @@ impl dyn Inode {
             return_errno!(Errno::EISDIR);
         }
 
-        let file_len = self.len();
-        if buf.len() < file_len {
-            buf.resize(file_len, 0);
+        let file_size = self.size();
+        if buf.len() < file_size {
+            buf.resize(file_size, 0);
         }
-        self.read_at(0, &mut buf[..file_len])
+        self.read_at(0, &mut buf[..file_size])
```

Comment:
I think `read_to_end` should be renamed to `reall_all`. Rust std's `Read` trait has a method called `read_to_end`. The name makes sense for the `Read` trait because the trait should be implemented on a position-aware reader. Like a normal file descriptor. However, our `Inode` trait does not maintain the current position for reading or writing. Thus, reading to end does not make sense for `Inode`.

Reply:
Done